deploy/scc.yaml

```diff
@@ -0,0 +1,34 @@
+apiVersion: v1
+kind: SecurityContextConstraints
```

Comment:
probably we just need a few of those, not all. But we can fine tune that in a follow up PR

Reply:
agree will do in follow up patch